telegram/ext/_application.py

```diff
@@ -379,12 +376,25 @@ def job_queue(self) -> Optional["JobQueue[CCT]"]:
             )
         return self._job_queue
 
+    @property
+    def update_processor(self) -> "BaseUpdateProcessor":
+        """:class:`telegram.ext.BaseUpdateProcessor`: The update processor used by this
+        application.
+
+        .. seealso:: :wiki:`Concurrency`
+
+        .. versionadded:: NEXT.VERSION
+        """
+        return self._update_processor
+
     async def initialize(self) -> None:
         """Initializes the Application by initializing:
 
         * The :attr:`bot`, by calling :meth:`telegram.Bot.initialize`.
         * The :attr:`updater`, by calling :meth:`telegram.ext.Updater.initialize`.
         * The :attr:`persistence`, by loading persistent conversations and data.
+        * The :attr:`update_processor` by calling
+            :meth:`telegram.ext.BaseUpdateProcessor.initialize`.
```

Comment:
```suggestion
        :meth:`telegram.ext.BaseUpdateProcessor.initialize`.
```

Reply:
Why? sphinx gives warning if I do so.

Reply:
it's not rendered properly, just align the `:meth:` with the `The`